announcements/2021/week09.md

```diff
@@ -0,0 +1,29 @@
+# :zap: This week in provider-independent announcements (March 5, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[NGINX IC v1.15.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.15.0/) upgrades the container images (controller container to v0.44.0 and kube-webhook-certgen to v1.5.1). It also removes conflicting admission webhook api versions, ensuring the the tree of apiversions will be valid according to the webhook.
```

Comment:
@QuentinBisson looks good to you? Seems Gerald is AFK haha

Reply:
Yes this works for me ;)

Reply:
thanks!